php/Terminus/Commands/SiteCommand.php

```diff
@@ -676,7 +676,11 @@ public function deploy($args, $assoc_args) {
 
     $workflow = $env->deploy($params);
     $workflow->wait();
-    $this->workflowOutput($workflow);
+    if ($workflow->get('result') == 'succeeded') {
+      $this->log()->info($workflow->get('active_description'));
+    } else {
+      $this->failure('Deployment failed.');
+    }
```

Comment:
Isn't this what the `$this->workflowOutput` function [does anyway](https://github.com/pantheon-systems/terminus/blob/2dfbb24c7eafbcb7783408dd5acc329967575465/php/Terminus/Commands/TerminusCommand.php#L181-L187)? It seems like maybe a platform issue that a failed deploy workflow doesn't have a meaningful error message.

Or, could we modify the `workflowOutput` method itself to pass in a custom error message? I just don't like repeating the logic of checking workflow success.